content/concepts/feathersjs.textile

```diff
@@ -268,4 +268,4 @@ The problem with adding load balancers and multiple servers is the associated co
 
 "Ably’s engineering blog":https://www.ably.io/blog/tag/ably-engineering/ describes problems and solutions associated with scaling realtime apps. When you find your (or your team’s) time taken up solving and/ or provisioning for these issues, that’s when - we believe - it’s worth looking for a cloud or better still serverless solution, leaving you free to concentrate on the core functionality of your app. That said, for small-scale applications or developers starting out with realtime - until realtime functionality scales and gets complicated, there is indeed a point at which FeathersJS could fit the bill of the holy grail of frameworks. Developers can judge for themselves when holy grail status starts to fade, and think about how to move on.
 
-*"Ably":https://www.ably.io/ is a global cloud network for streaming data and managing the full lifecycle of realtime APIs. To talk about scaling your app or service, get in touch with our dev-rel team.*
\ No newline at end of file
+*"Ably":https://www.ably.io/ is a global cloud network for streaming data and managing the full lifecycle of realtime APIs. To talk about scaling your app or service, get in touch with our dev-rel team.*
```

Comment:
I realise this is not part of the original ticket, but I see this is out of date and that we're asking people to contact the Dev Rel team but provide no contact link. Could you replace this paragraph with:

"Ably":https://ably.com is a realtime messaging platform. We deliver billions of realtime messages every day to more than 50 million end-users across web, mobile, and IoT platforms.

Developers use Ably’s 40+ SDKs to build realtime capabilities in their apps with our "multi-protocol pub/sub messaging":https://ably.com/channels, "presence":https://www.ably.io/documentation/core-features/presence, "push notifications":https://ably.com/push-notifications, "free streaming data sources":https://ably.com/hub from across industries like transport and finance, and "integrations that extend Ably":https://ably.com/reactor into third-party clouds and systems like AWS Kinesis and RabbitMQ.

To talk about scaling your realtime capabilities, get in touch with our "Developer Relations team":https://ably.com/contact.

Reply:
Sure, no problem